lib/rules/lint-no-curly-component-invocation.js

```diff
@@ -31,8 +32,40 @@ const IN_BUILT_HELPERS = [
   'with',
   'yield',
 ];
+const DEFAULT_CONFIG = {
+  allow: [],
+  disallow: [],
+  requireDash: true,
```

Comment:
Note for future: I'd like to change this to `false` in the next major bump (which will be focused on Octane support).